css/CalendarMonthGrid.scss

```diff
@@ -3,6 +3,7 @@ $react-dates-width-day-picker: 300px;
 .CalendarMonthGrid {
   background: #fff;
   z-index: 0;
+  text-align: initial;
```

Comment:
Since initial doesn't work in all browsers we support ( http://caniuse.com/#search=initial ) could we also add, above this, `text-align: left;`, so that this bug can be fixed in those browsers too?